src/components/LinkDialog-css.ts

```diff
@@ -0,0 +1,4 @@
666;text-decoration:underline}.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel{position:absolute;-webkit-appearance:none;display:flex;align-items:center;justify-content:center;top:24px;right:24px;width:24px;height:24px;border-radius:12px;background-color:#e7e7e7;cursor:pointer}.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel-x{position:relative;display:block}.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel-x::before,.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel-x::after{content:"";position:absolute;display:block;top:-1px;left:-7px;width:14px;height:2px;background-color:#999;transition:background-color .2s}.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel-x::before{transform:rotate(45deg)}.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel-x::after{transform:rotate(135deg)}.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel:hover .-cbwsdk-link-dialog-box-cancel-x-a,.-cbwsdk-css-reset .-cbwsdk-link-dialog-box-cancel:hover .-cbwsdk-link-dialog-box-cancel-x-b{background-color:#000}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container{display:block}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-hidden{display:none}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box{background-color:#2a2a2a}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content h3{color:#ccc}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content-qrcode{background-color:#2a2a2a;box-shadow:4px 4px 8px rgba(0,0,0,.5),-8px -8px 8px #343434}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content-qrcode>p{color:#999}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content-qrcode-connecting{background:rgba(42,42,42,.98)}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content-qrcode-connecting p{color:#ddd}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content a,.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content a:link,.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content a:visited{color:#888}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content a:hover,.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-content a:active{color:#aaa}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-cancel{background-color:#333}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-cancel-x::before,.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-cancel-x::after{background-color:#aaa}.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-cancel:hover .-cbwsdk-link-dialog-box-cancel-x::before,.-cbwsdk-css-reset .-cbwsdk-link-dialog-container-dark .-cbwsdk-link-dialog-box-cancel:hover .-cbwsdk-link-dialog-box-cancel-x::after{background-color:#eee}`
```

Comment:
these look generated, why are they checked in?

Reply:
Do you recommend running `compile-assets.js` as part of the testing step instead?

Reply:
Absolutely yes. Generated files should never be committed to version control.